lib/modules/platform/github/index.ts

```diff
@@ -628,15 +628,6 @@ async function getOpenPrs(): Promise<PrList> {
         delete pr.headRefName;
         pr.targetBranch = pr.baseRefName;
         delete pr.baseRefName;
-        // https://developer.github.com/v4/enum/mergeablestate
-        const canMergeStates = ['BEHIND', 'CLEAN', 'HAS_HOOKS', 'UNSTABLE'];
-        const hasNegativeReview = pr.reviews?.nodes?.length > 0;
-        // istanbul ignore if
-        if (hasNegativeReview) {
-          pr.cannotMergeReason = `PR has a negative review`;
-        } else if (!canMergeStates.includes(pr.mergeStateStatus)) {
-          pr.cannotMergeReason = `pr.mergeStateStatus = ${pr.mergeStateStatus}`;
-        }
```

Comment:
So this is no longer possible:
https://github.com/renovatebot/renovate/blob/a61821bde86356cde61d56e2ab2373366af01cb1/lib/workers/repository/update/pr/automerge.ts#L57-L65

🤔

Reply:
Yes, it was intentional. The reason is that automerge won't work anyway, but we don't have to request additional PR data via REST, once GraphQL requests aren't used anymore for pull request fetching.

Reply:
It's going to cost us an extra query per PR, so we may as well use that query "lazily" by attempting to automerge instead of proactively checking it for every PR

Reply:
I'm also pretty sure that our git layer plus status checks will catch nearly all cases anyway